src/sql/workbench/contrib/query/browser/query.contribution.ts

```diff
@@ -447,6 +447,11 @@ const queryEditorConfiguration: IConfigurationNode = {
 			'description': localize('queryEditor.results.showActionBar', "Whether to show the action bar in the query results view"),
 			'default': true
 		},
+		'queryEditor.results.copyHeaders.delimeter': {
+			'type': 'string',
+			'description': localize('queryEditor.results.copyHeaders.delimeter', "The custom delimiter to use between result headers when copying"),
+			'default': '\t'
```

Comment:
How does this show up in the UI? Is it clear that it's a tab?

Reply:
No, it's not clear. It shows up as any tab would in a text document. Is there a way to make it much more clear?

Reply:
Maybe making this setting a boolean type instead of a string type would help to clarify that. The tab setting can be the default, and checking the box would set copying to delimit by comma?